Description: Disable application modules by default
 Below is a list of application modules we have disabled by default. Mostly
 because we don't want to support the modules.
 .
 * app_adsiproj.so
 * app_alarmreceiver.so
 * app_dictate.so
 * app_externalivr.so
 * app_festival.so
 * app_followme.so
 * app_getcpeid.so
 * app_ices.so
 * app_image.so
 * app_minivm.so
 * app_morsecode.so
 * app_mp3.so
 * app_nbscat.so
 * app_sendtext.so
 * app_sms.so
 * app_test.so
 * app_url.so
Forwarded: not-needed
Author: Paul Belanger <paul.belanger@polybeacon.com>
Last-Updated: 2012-02-08
--- a/apps/app_adsiprog.c
+++ b/apps/app_adsiprog.c
@@ -26,6 +26,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<depend>res_adsi</depend>
 	<support_level>extended</support_level>
  ***/
--- a/apps/app_alarmreceiver.c
+++ b/apps/app_alarmreceiver.c
@@ -30,6 +30,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_dictate.c
+++ b/apps/app_dictate.c
@@ -28,6 +28,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_externalivr.c
+++ b/apps/app_externalivr.c
@@ -32,6 +32,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_followme.c
+++ b/apps/app_followme.c
@@ -29,6 +29,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<depend>chan_local</depend>
 	<support_level>core</support_level>
  ***/
--- a/apps/app_getcpeid.c
+++ b/apps/app_getcpeid.c
@@ -26,6 +26,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_ices.c
+++ b/apps/app_ices.c
@@ -28,6 +28,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
  
--- a/apps/app_image.c
+++ b/apps/app_image.c
@@ -26,6 +26,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
  
--- a/apps/app_morsecode.c
+++ b/apps/app_morsecode.c
@@ -25,6 +25,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_mp3.c
+++ b/apps/app_mp3.c
@@ -29,6 +29,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
  
--- a/apps/app_nbscat.c
+++ b/apps/app_nbscat.c
@@ -26,6 +26,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
  
--- a/apps/app_sendtext.c
+++ b/apps/app_sendtext.c
@@ -28,6 +28,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>core</support_level>
  ***/
  
--- a/apps/app_sms.c
+++ b/apps/app_sms.c
@@ -35,6 +35,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_test.c
+++ b/apps/app_test.c
@@ -28,6 +28,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_url.c
+++ b/apps/app_url.c
@@ -26,6 +26,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
  
--- a/apps/app_minivm.c
+++ b/apps/app_minivm.c
@@ -142,6 +142,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
--- a/apps/app_festival.c
+++ b/apps/app_festival.c
@@ -28,6 +28,7 @@
  */
 
 /*** MODULEINFO
+	<defaultenabled>no</defaultenabled>
 	<support_level>extended</support_level>
  ***/
 
